File: BOJ/8958.cpp
#include <stdio.h>
int main() {
    int tc;
    char str[81];
    scanf("%d", &tc);
    while (tc > 0) {
        int sum = 0, cnt = 0;
        scanf("%s", str);

        for (int i = 0; str[i]; i++) {
            str[i] == 'O' ? cnt++ : cnt = 0;
            sum += cnt;
        }
        tc--;
    printf("%d\n", sum);
    }
}

File: BOJ/8958.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
for (let i = 1; i <= parseInt(input[0]); i += 1) {
    let sum = 0;
    let cnt = 0;
    for (let j = 0; j <= input[i].length; j += 1) {
        input[i].charAt(j).toUpperCase() === 'O' ? cnt += 1 : cnt = 0;
        sum += cnt;
    }
    console.log(sum);
}
